### ISSUE TITLE
Selected state for TagView

### ISSUE THREAD

**Issue by Ralphilius** [whilu/AndroidTagView#12]
I wonder if we can set selected state when we click on a TagView. Or at least, should we have `getTagView(int position)` method in the `TagContainerLayout` and an `attrs` to set `TagView`'s background?

**Comment by whilu** [comment]
Yeah, put some kind of state for ```TagView``` is a good suggestion!  I think I will add this feature in next version. And there are some methods to get ```TagView```'s attribute, like ```getTagText(int position)``` method return the string text in a ```TagView```,  so I think it's not necessary to add ```getTagView(int position)``` method,  and add an ```attr``` to set ```TagView```'s background, you mean set all ```TagView```'s background in one ```TagContainerLayout```?

**Comment by Ralphilius** [comment]
> add an `attr` to set `TagView`'s background, you mean set all `TagView`'s background in one `TagContainerLayout`?

Yes. That is what I suggest. Since `TagView`s are dynamically added to `TagContainerLayout` it is difficult to add different state color for each `TagView`. I have a local fork that has such functionality. If you want I can create a PR.

**Comment by whilu** [comment]
Welcome to create PR, it's great! Thank you!
2016年5月1日 上午3:33，"Ralphilius" <notifications@github.com>写道：

**Comment by SanyasiraoM** [comment]
@Ralphilius @whilu Any update on this? Thank you

**Comment by csbenz** [comment]
@Ralphilius did you submit the PR?
